### PR TITLE
add endpoint to get user by id (uuid)

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/models/remote_provider.go [hunk 1]
```diff
@@ -308,6 +308,37 @@ func (l *RemoteProvider) fetchUserDetails(tokenString string) (*User, error) {
...
+	if err != nil {
+		logrus.Errorf("unable to read response body: %v", err)
+		return nil, err
+	}
+
+	if resp.StatusCode == http.StatusOK {
+		logrus.Infof("user profile successfully retrieved from remote provider")
+		return bdr, nil
+	}
+	logrus.Errorf("error while fetching user profile: %s", bdr)
```
Comment (leecalcote) [inline server/models/remote_provider.go:338]: Are MeshKit Errors not being used here?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Lint items to attend to

**Comment by Chadha93** [comment]
https://github.com/layer5labs/meshery-extensions/pull/1638

File: server/models/remote_provider.go (hunk 1 above)
Comment (leecalcote) [reply server/models/remote_provider.go:338]: @MUzairS15 an item to consider.

**Review by Chadha93 — APPROVED** [review]
LGTM, Thank @MUzairS15

**Pull request merged**